generated/python/grpc-google-cloud-functions-v1beta2/setup.py

```diff
@@ -38,6 +38,6 @@
   install_requires=install_requires,
   license='Apache-2.0',
   packages=find_packages(),
-  namespace_packages=['google.cloud.grpc.functions', 'google.cloud.grpc', 'google.cloud', 'google'],
+  namespace_packages=['google.cloud.grpc.functions', 'google.cloud.grpc', 'google.cloud.proto.functions', 'google.cloud.proto', 'google.cloud', 'google'],
```

Comment:
This doesn't look right: `google.cloud.grpc.*` shouldn't still be here, since it was renamed to `google.cloud.proto.*`. The `google/cloud/grpc` directory also should have been removed.

When you re-run the generation, do you still see that directory (and these packages declared in `setup.py` even if you clean `artman/output` beforehand?